bot/Bots/QnABot.cs

```diff
@@ -43,7 +46,10 @@ protected override async Task OnMembersAddedAsync(IList<ChannelAccount> membersA
             {
                 if (member.Id != turnContext.Activity.Recipient.Id)
                 {
-                    await turnContext.SendActivityAsync(MessageFactory.Text($"Hello and welcome!"), cancellationToken);
+                    var GreetingMsg = !string.IsNullOrEmpty(this._configuration["GreetingMsg"]) ?
+                    this._configuration["GreetingMsg"] : $"Hello and welcome!";
+
+                    await turnContext.SendActivityAsync(MessageFactory.Text(GreetingMsg), cancellationToken);
```

Comment:
```suggestion
                    var greetingMsg = this._configuration["GreetingMsg"] ?? $"Hello and welcome!";

                    await turnContext.SendActivityAsync(MessageFactory.Text(greetingMsg), cancellationToken);
```

Updates to make local var lowerCase and using ull-coalescing to simplify logic similar to [DefaultAnswer config check](https://github.com/VermontDepartmentOfHealth/covid-bot/blob/master/bot/Dialog/QnAMakerBaseDialog.cs#L68)